backend.native/tests/interop/basics/cmacros.def

```diff
@@ -10,5 +24,19 @@
 #define ONE_POINT_ZERO 1.0
 #define ONE_POINT_FIVE 1.5f
 
+#define NULL_PTR (void*)0
```

Comment:
It makes sense to add brackets to this and following definitions to make them less ambiguous, i.e.
`#define NULL_PTR ((void*)0)`

Reply:
Fixed.